test/Agent-test.js

```diff
@@ -0,0 +1,47 @@
+import chai from 'chai';
+const expect = chai.expect;
+
+import Agent from '../src/Agent'
+import Trip from '../src/Trip'
+import trips from './trips-test-data'
+import destinations from './destinations-test-data'
+import Destination from '../src/Destination'
+
+const destinationsArray = destinations.destinations.map(destinationObject => new Destination(destinationObject))
+
+const tripsArray = trips.trips.map(tripObject => new Trip(tripObject, destinationsArray))
+
+describe('Trip', function() {
+  let agent
+
+  beforeEach(() => {
+    agent = new Agent()
+  });
+
+  it('should be a function', function() {
+    expect(Agent).to.be.a('function')
```

Comment:
I'm never sure how I feel about set up tests. (ie instanceOf and to.be.a('dataType')) It makes sense, to cover all your basis, but it still smells like there's a sense of redundancy in it.